flash/text/classification/tokenizers/transformers.py

```diff
@@ -0,0 +1,83 @@
+# Copyright The PyTorch Lightning team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from typing import Any, List, Mapping, Sequence, Tuple, Union
+
+import torch
+from pytorch_lightning.utilities.exceptions import MisconfigurationException
+
+from flash.core.utilities.imports import _TEXT_AVAILABLE
+from flash.text.classification.tokenizers.base import BaseTokenizer
+
+if _TEXT_AVAILABLE:
+    from transformers import AutoConfig, AutoTokenizer
+
+
+class TransformerTokenizer(BaseTokenizer):
+    def __init__(self, backbone: str, pretrained: bool = True, **backbone_kwargs):
+        super().__init__(backbone, pretrained)
+
+        self.tokenizer = AutoTokenizer.from_pretrained(backbone)
```

Comment:
Should this be an HF Transformer ?

Reply:
No, because this is the module in charge of loading the tokenizers. If the nomenclature is confusing, I can change it.